COMETwebapp/Components/UserManagement/UserManagementTable.razor

```diff
@@ -45,7 +45,17 @@ Copyright (c) 2023 RHEA System S.A.
         <DxGridDataColumn FieldName="@nameof(PersonRowViewModel.Role)" Caption="Role" MinWidth="80" SearchEnabled="false" />
         <DxGridDataColumn FieldName="@nameof(PersonRowViewModel.IsActive)" UnboundType="GridUnboundColumnType.Boolean" MinWidth="80" SearchEnabled="false">
             <CellDisplayTemplate>
-                <DxCheckBox Id="activatePerson" CssClass="d-inline-block" CheckedChanged="@((bool value) => this.ViewModel.ActivatingOrDisactivatingPerson(context, value))" Checked="(bool)context.Value" Enabled="this.ViewModel.IsAllowedToWrite" />
```

Comment:
Maybe, the IsAllowedToWrite could be set to false on the VM directly when it is the activePerson directly?

Reply:
But IsAllowedToWrite is not created for each row, it's for all rows